### ISSUE TITLE
Add example for cross entropy loss

### ISSUE THREAD

**Issue by jaiabhayk** [FalkonML/falkon#4]
Hi,

The present documentation has only example related to binary cross entropy loss (logistic regression). I was wondering would it be possible to include example for cross entropy loss (for more than two classes) ?

**Comment by Giodiro** [comment]
For multiple classes you can follow two approaches:
 - Use the normal (least squares) falkon with one-hot encoded targets: see [this notebook](https://falkonml.github.io/falkon/examples/hyperparameter_tuning.html) for an example
 - Use the logistic falkon estimator for every pair of classes (you can do one vs. all or one vs. one for multiclass).

In general, since logistic falkon is quite a bit slower than the normal one, I would just use the normal one with one-hot encoded targets.
